spring-cloud-function-samples/spring-cloud-function-sample-compiler/src/test/java/com/example/SampleApplicationTests.java

```diff
@@ -41,8 +41,8 @@
 	@Test
 	public void lowercase() {
```

Comment:
i know this was already incorrect, but just noticing this method is not about "uppercase"